fuse/core.go

```diff
@@ -92,13 +92,8 @@ func (f *fuseNode) applyAttr(a *fuse.Attr, attr *plugin.EntryAttributes, default
 		a.Mode = defaultMode
 	}
 
-	const blockSize = 4096
 	if attr.HasSize() {
```

Comment:
We could set the size here using the cached content, but I'd rather we not do that unless we need to. This maintains attributes-symmetry between FUSE and API. Otherwise, API would report unknown size, but FUSE would (sometimes) report a size, which is kind of weird.